coala_quickstart/interaction/Logo.py

```diff
@@ -0,0 +1,60 @@
+from coala_quickstart.Strings import coala_bear_logo, welcome_messages
+
+
+def text_wrap(*args, delimiter=' ', end='\n', limit=80):
+    """
+    Breaks it wordwise when going further than ``limit`` chars.
+
+    :param args:
+        The stuff you want to have printed.
+    :param delimiter:
+        Will be placed between all the args.
+    :param limit:
+        Char limit.
+    :param end:
+        An ending, will be put as last character and doesn't count into the
+        char limit.
+    :return:
+        A list of strings, each element to be written on its own line.
+    """
+    output = delimiter.join(args)
+    lines = output.splitlines(keepends=True)
+    results = [""]
+    for line in lines:
+        curr_print = line
+        while len(curr_print.rstrip('\n')) > limit:
+            splitpos = curr_print[:limit].rfind(' ')
```

Comment:
You may want to file it here: https://gitlab.com/sims1253/LoneCoffin/blob/master/utils.py

I just copied it (with a few smalll modifications)